WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalErrorRetryTableViewCell.swift

```diff
@@ -0,0 +1,142 @@
+
+import UIKit
+
+import Gridicons
+import WordPressShared
+
+// MARK: - VerticalErrorRetryTableViewCellAccessoryView
+
+/// The accessory view comprises a retry Gridicon
+private class VerticalErrorRetryTableViewCellAccessoryView: UIStackView {
+
+    // MARK: Properties
+
+    /// A collection of parameters uses for view layout
+    private struct Parameters {
```

Comment:
A bit of a nitpick, but I'd maybe suggest a slightly more descriptive name for this – I think we've used something like `Metrics` in other places. Or `Dimensions` or `Appearance`?

Reply:
I've seen a mix of this in use in the app. I don't feel strongly about it, given that it's private. I pushed a commit that reflects this change.